cmd/kritis/admission/main.go

```diff
@@ -82,20 +95,28 @@ func NewServer(addr string) *http.Server {
 }
 
 func StartCronJob() error {
-	checkInterval, err := time.ParseDuration(cronInterval)
+	d, err := time.ParseDuration(cronInterval)
 	if err != nil {
 		return err
 	}
 	ctx := context.Background()
```

Comment:
Move this to right under where it is used, or better yet, inline it:

go cron.Start(context.Background(), *config, d)